modules/oe_bootstrap_theme_helper/tests/themes/oe_bootstrap_theme_test_subtheme1/oe_bootstrap_theme_test_subtheme1.info.yml

```diff
@@ -6,3 +6,4 @@ core_version_requirement: ^10
 openeuropa:
   bootstrap_component_library:
     components_location: components/templates
+    bcl_icon_path: 'assets/icons/bcl-default-icons-test.svg'
```

Comment:
We don't need this here if we create test themes.

Reply:
done.